spec/datasets/djornl/definitions.yaml

```diff
@@ -21,20 +21,31 @@ definitions:
       pattern: ^(\S+__){3}(\S+)$
     _from:
       type: string
-      title: Gene ID
+      title: Node ID
+      format: regex
+      pattern: ^\S{2,}.*$
     _to:
       type: string
-      title: Gene or Phenotype ID
+      title: Node ID
+      format: regex
+      pattern: ^\S{2,}.*$
```

Comment:
Do these need to be single quoted so the backslashes aren't handled as escape chars? 
Assuming you tested this so I'm guessing not

Reply:
This was weirdly hard to find, but according to this page:
http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html
In unquoted strings, backslashes do not need to be escaped